modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/SecondaryRecipients.tsx

```diff
@@ -30,6 +31,14 @@ interface SecondaryRecipientsProps {
 	values: NotificationTemplate;
 }
 
+export function handleRecipientTypeValue(newRecipientTypeValue: string) {
```

Comment:
Hey @vngoliveira, what do you think about bringing the idea of ​​cleaning the values ​​to the name of this function?